src/Entity/Link.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Entity;
+
+class Link
+{
+    public function __construct(
+        public readonly string $id,
+        public readonly string $link,
+        public readonly ?string $title,
+        public readonly array $tags,
```

Comment:
Как уже писал выше, лучше сделать как-то так
```suggestion
        public readonly string $long_url = '',
        public readonly ?string $title = null,
        public readonly array $tags = [],
```